nepomuk/src/main/java/edu/asu/diging/gilesecosystem/nepomuk/core/service/impl/RequestProcessor.java

```diff
@@ -126,6 +138,35 @@ public void processRequest(IStorageRequest request) {
         }
     }
     
+    @Override
+    public void processRequest(IStorageDeletionRequest request) {
```

Comment:
let's put this in its own class. Single responsibility principle, every class should only be responsible for one thing (either initiating storage or deleting a file).